### DISCUSSION TITLE
How to show empty state when No items found.

### DISCUSSION THREAD

**Discussion by dhruva81** [rappasoft/laravel-livewire-tables#670] [Q&A]
I want to show custom view/empty state, when there is no record found.

<img width="341" alt="image" src="https://user-images.githubusercontent.com/42173738/155768312-df3219c0-8a25-49d1-b96c-0631591b8b41.png">

Kindly guide me, how can I achieve this type of custom view for empty state.

Thanks

**Comment by lrljoe** [comment]
[https://rappasoft.com/docs/laravel-livewire-tables/v3/datatable/available-methods#content-setemptymessage](https://rappasoft.com/docs/laravel-livewire-tables/v3/datatable/available-methods#content-setemptymessage)

If you'd rather a view, then please open a Feature Request
